src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/CourtCaseEventsService.kt

```diff
@@ -32,6 +27,11 @@ class CourtCaseEventsService(
       is InvalidPNCIdentifier -> trackEvent(INVALID_PNC, mapOf("PNC" to pncIdentifier.invalidValue()))
       else -> trackEvent(MISSING_PNC, emptyMap())
     }
+    if (person.otherIdentifiers?.croIdentifier != null) {
+      if (!person.otherIdentifiers.croIdentifier.valid) {
+        trackEvent(INVALID_CRO, mapOf("CRO" to person.otherIdentifiers.croIdentifier.croId))
```

Comment:
I think this will just log an empty string if it's invalid, if I have understood `CroIdentifier` correctly. Let's put a test in CourtCaseEventsServiceIntTest to verify this.

More generally though:
I am not sure why we care about invalid CROs really - seems a lot of complexity for us to log this event. What are we proposing to do with this information?
Same goes for PNCs now - I suggest we could ditch a lot of the complexity there now we have the validation working.